cloudbuild.yaml

```diff
@@ -0,0 +1,14 @@
+steps:
+- name: maven:3.6.0-jdk-11-slim
+  entrypoint: 'mvn'
+  args: ['install']
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
We don't have a Dockerfile... why is there a 'docker build' step?

What I'd expect, instead, is a 'gcr.io/cloud-builders/gcloud' step to perform a deployment to App Engine (which you might do with Maven, directly, so you probably only need one step).